### ISSUE TITLE
Impossible to open popup in Chrome

### ISSUE THREAD

**Issue by panpiotrs** [bitwarden/browser#604]
The issue started with some recent update (when new icons and new layout for adding login was introduced). It happens mainly when there's a lot logins suggested for current site. In example below there's 19 logins (for different subdomains). Sometimes it happens even when there's only one match for a page, but in those cases after couple clicks it shows.
Chrome 66, Mac OS 10.13
No problems in Firefox.

![](https://i.imgur.com/dROzFYZ.gif)

**Comment by kspearrin** [comment]
Hmm. What if you click it and wait a few seconds, does it eventually open up?

**Comment by panpiotrs** [comment]
No, when I do that the small square stays there all the time.

**Comment by kspearrin** [comment]
Can you inspect it and see if there is some error in the dev console?

**Comment by panpiotrs** [comment]
Do you mean Chrome Dev Tools? Nothing there in the console.
You can't see it on the gif, but each time when I click on the icon the normal sized popup opens for a few milliseconds and it collapses to the small square.

**Comment by kspearrin** [comment]
Yes, right click the small window and inspect.

**Comment by panpiotrs** [comment]
No, there are no errors, the source also looks fine, everything seems to be loaded. I guess it thinks it's opened, but somehow Chrome make it smaller then it should be.
I also have Vivaldi, no problems there.

**Comment by kspearrin** [comment]
What is your screen resolution?

**Comment by panpiotrs** [comment]
1680x1050

**Comment by panpiotrs** [comment]
Also no problems in Chrome on Linux Mint, so it's only Chrome on Mac issue for me.

**Comment by kspearrin** [comment]
It seems something in Chrome 66 broke this. I was using Chrome 65 on my macbook and there were no problems. I just updated to Chrome 66 and now I can replicate the problem.

**Comment by kspearrin** [comment]
Looks to be a known bug in Chrome that they supposedly have fixed in Chrome 67 ... 🤷‍♂️ 

See https://bugs.chromium.org/p/chromium/issues/detail?id=428044#c44

**Comment by panpiotrs** [comment]
Oh, Chrome 66 was released 6 days ago, the same day you released new version of the extension, this is why it looked like the new layout broke it :) good to know, thanks for the info :)

**Comment by haveneersrobin** [comment]
Jep, same thing happening to me. Sometimes pressing it a second time will actually open it 


![kapture 2018-05-10 at 10 53 01](https://user-images.githubusercontent.com/7559898/39861262-57c43f44-5440-11e8-84b0-9d92fead2112.gif)

**Comment by kidk** [comment]
Can confirm that this indeed does not seem to happen anymore in Chrome 67.
